README.md

```diff
@@ -100,6 +100,14 @@ yarn bundle:chrome
 yarn bundle:firefox
 ```
 
+### Release a new version
+1. Tick the version with `yarn version` https://yarnpkg.com/en/docs/cli/version
+2. Push the tags with `git push --tags`
```

Comment:
💡 I didn't know Yarn creates Git tags. Nice.

I was slightly surprised though when reading through the steps.
Maybe append `… (creates a Git tag)` to the first step?